Cargo.toml

```diff
@@ -30,7 +30,7 @@ cstr = "0.2"
 libc = "0.2"
 bincode = "1"
 serde = { version = "1", features = ["derive"] }
-nix = { version = "0.25", default-features = false, features = ["fs", "mount", "user"] }
+nix = { version = "0.26.1", default-features = false, features = ["fs", "mount", "user"] }
 which = { version = "4", optional = true }
```

Comment:
Is the .1 necessary?

Reply:
Also, does the version of the fuse3 crate need to be bumped to 0.5.2?

Reply:
The .1 is probably not necessary.  But 0.26.0 was a mistake (my mistake) and really shouldn't be used anywhere.
The fuse3 crate's version doesn't need to be bumped until a release is imminent.  I prefer not to bump it preemptively, because that makes it harder to use a `[patch.crates-io]` section in your `Cargo.toml` file to get the latest version from git.

Reply:
OK, we'd like to use this crate, but we need this change in released version so I was hoping we could get a release made with this nix version bump.